vnext/Microsoft.ReactNative/IJSValueReader.idl

```diff
@@ -17,12 +19,12 @@ namespace Microsoft.ReactNative {
   // Forward only reader for JSON like streams or trees.
   [webhosthidden]
   interface IJSValueReader {
-    JSValueType ValueType { get; };
-    Boolean GetNextObjectProperty(out String propertyName);
-    Boolean GetNextArrayItem();
-    String GetString();
-    Boolean GetBoolean();
-    Int64 GetInt64();
-    Double GetDouble();
+    [noexcept2] JSValueType ValueType { get; };
```

Comment:
>noexcept2 [](start = 5, length = 9)

why noexcept2 instead of just noexcept?

Reply:
The noexcept is the reserved keyword in future versions of MIDL. We cannot use it yet. Though, the same functionality can be provided by a custom attribute for now. Thus, we should have a different name and I chose the noexcept2.

---
In reply to: [368188593](https://github.com/microsoft/react-native-windows/pull/3905#discussion_r368188593) [](ancestors = 368188593)